src/AutoMerge/util.jl

```diff
@@ -84,7 +84,7 @@ function parse_registry_pkg_info(registry_path, pkg, version=nothing)
     return (; uuid=uuid, repo=repo, subdir=subdir, tree_hash=tree_hash)
 end
 
-function _comment_disclaimer()
+function _comment_disclaimer(point_to_slack::Bool=false)
```

Comment:
It would probably be best to make this a keyword argument.

Reply:
Sure! Do we want to show the message both for new packages and new versions? Right now it's only for new versions, but I can easily add the flag for the `comment_text_fail(::NewVersion, ...` method as well

Reply:
Let's do it for both.

Reply:
Do you want it to be a keyword one level up, in `comment_text_fail`, as well? That function already has `suggest_onepointzero` as a positional argument, which seems similar. But adding `point_to_slack` replaces `version` as the fourth positional argument there. So, make `point_to_slack` a keyword argument after `version`?

Reply:
Okay, I made it a keyword argument throughout in the latest commit https://github.com/JuliaRegistries/RegistryCI.jl/pull/498/commits/4d5591ca9066bc1c41d23a43ad9b96e132cc2faa, but I can roll that back or change it if you have something different in mind.

Reply:
Yeah, in hindsight, `suggest_onepointzero` should have also been a kwarg.